### ISSUE TITLE
ActivityIndicator Not Working in Moto G4 Plus running Android 7.0

### ISSUE THREAD

**Issue by nsreenath** [facebook/react-native#18229]
ActivityIndicator is not working in Moto G4 Plus running Android 7.0.
Did a fresh clone of React Native repo on March 7th, 2018 on Windows 10 machine and Tested with [RNTester](https://github.com/facebook/react-native/blob/master/RNTester) which is installed on phone as per instructions in its [README](https://github.com/facebook/react-native/blob/master/RNTester/README.md).
### Environment

`react-native info` is not working on cloned react native repo because it is not a react-native project.
But this is the info from another project:
  OS: Windows 10
  Node: 7.5.0
  Yarn: 0.20.3
  npm: 4.1.2

### Expected Behavior
ActivityIndicator should be visible on screen.

### Actual Behavior

ActivityIndicator is not visible on screen.
![screen](https://user-images.githubusercontent.com/3154529/37056404-58b55f50-21aa-11e8-9126-b790acebc1af.png)

### Steps to Reproduce

Run RNTester app, or any react native app that displays an ActivityIndicator, on Moto G4 Plus running Android 7.0

Issue #11964 seems to be this same issue, but it is closed without attending the issue.

**Comment by nsreenath** [comment]
ActivityIndicator is also not displayed in Expo APIs component showcase in latest Expo app for Android.

![screenshot_20180307-090707](https://user-images.githubusercontent.com/3154529/37072353-f369a0ec-21e6-11e8-87b8-7f711dda402f.png)

**Comment by nsreenath** [comment]
The issue seem to got fixed after restarting the phone.

I restarted my phone few days after I raised the issue. When I checked the same [RNTester ](https://github.com/facebook/react-native/blob/master/RNTester) app again today without updating the app, the `ActivityIndicator` is working. The day when I raised the issue, `ActivityIndicator` was not working even when running `Expo APIs` on `Expo client for Android`. 

So it could either be an issue with React Native or an issue with the OS.
